src/component/handlers/edit/DraftEditorEditHandler.ts

```diff
@@ -23,11 +23,12 @@ import {editOnKeyDown} from './editOnKeyDown';
 import editOnPaste from './editOnPaste';
 import editOnSelect from './editOnSelect';
 
-const isChrome = UserAgent.isBrowser('Chrome');
+const isChromium =
+  UserAgent.isBrowser('Chrome') || UserAgent.isBrowser('Electron');
 const isFirefox = UserAgent.isBrowser('Firefox');
 
 const selectionHandler: (e: DraftEditor) => void =
-  isChrome || isFirefox
+  isChromium || isFirefox
```

Comment:
This was the specific issue causing our bug. We want to register `editOnSelect` for onKeyUp on Chromium - see the comment in the code below.

Reply:
Nice! Good find.